modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/oauth/ocs/PlatformManagedOAuthConnectionProvider.java

```diff
@@ -389,4 +390,12 @@ public void onReturn(C connection) {
     }
   }
 
+  @Override
```

Comment:
If there is a default implementation, we don't need to repeat this code in the implementations.